bolt-design-system/bolt
# Blockquote | FE | Do not merge: Add prop to hide quotation marks

## Jira

http://vjira2:8080/browse/BDS-1593

## Summary

Add `no-quotes` prop to blockquote to hide quotation marks.

## Details

Until language-specific quotes are finished, users need to be able to hide quotes and include them directly in the content instead. The `no-quotes` prop gives them that ability.

## How to test

1. Review code changes.
2. Run branch locally and view [No Quotes Page](http://localhost:3000/pattern-lab/patterns/02-components-blockquote-25-blockquote-quote-variation/02-components-blockquote-25-blockquote-quote-variation.html).
3. Verify the quotation marks are hidden.
4. Verify quotes are displaying normally on other demos.
